Sources/AsyncStream.swift

```diff
@@ -1,7 +1,7 @@
 public protocol AsyncStream {
     var closed: Bool { get }
     func close() -> Bool
-    func receive(length length: Int, result: (Void throws -> Data) -> Void)
-    func send(data data: Data, result: (Void throws -> Void) -> Void)
+    func receive(upTo byteCount: Int, result: (Void throws -> Data) -> Void)
```

Comment:
@tannernelson @LoganWright  hahaha yeah I proposed that in our slack discussion about this. We thought it wouldn't be needed as the function signature has `byteCount` in it.

```
func receive(upTo byteCount: Int, result: (Void throws -> Data) -> Void)
```

But we also discussed how that would only be accessible by someone with an IDE. And adding `byte`, `bytes`, `kilobyte`, etc.. would make things much more readable. The argument @LoganWright used is that no one would use it. I said I would haha, and we should encourage it. Another problem @LoganWright pointed, is that having `bytes` on `Int` could clash with code from other frameworks. My view is if we worry too much about this we'll get too constrained. It's hard (maybe just impossible) to adapt to *everybody*. But also, I don't think `bytes` are 100% needed, so we could go either way. Also, those extensions could be added in another library if one really wants them.